app/services/plan_service/api/plans.rb

```diff
@@ -53,10 +53,7 @@ def get_current(community_id:)
 
     def expired?(community_id:)
```

Comment:
Good point. Yeah, apparently not, and apparently there's actually no need for `expired?` method.